Settings.php

```diff
@@ -185,4 +193,29 @@ private function createTrackToCustomSiteIdSetting()
         $this->addSetting($this->customPiwikSiteId);
     }
 
+    private function anonymizeCustomPiwikCheckbox()
+    {
+        $this->anonymizeCustomPiwik = $this->anonymizationOption("anonymizeCustomPiwik");
+
+        $this->addSetting($this->anonymizeCustomPiwik);
+    }
+
+    private function anonymizeSelfPiwikCheckbox()
+    {
+        $this->anonymizeSelfPiwik = $this->anonymizationOption("anonymizeSelfPiwik");
+
+        $this->addSetting($this->anonymizeSelfPiwik);
+    }
+
+    private function anonymizationOption($optionName)
+    {
+        $optionCheckbox = new SystemSetting($optionName, 'Turn on anonymization');
+        $optionCheckbox->type = static::TYPE_BOOL;
+        $optionCheckbox->uiControlType = static::CONTROL_CHECKBOX;
+        $optionCheckbox->description = 'Enable/Disable anonymization';
+        $optionCheckbox->defaultValue = true;
```

Comment:
Could you add a test that makes sure that by default anonymization is enabled? This would be quite important since "Anonymous" is in the plugin name so we need to make sure that this will be the case by default